src/main/webapp/js/captions.js

```diff
@@ -41,6 +57,53 @@ function execute(url) {
         return;
     }
 
+    // displays the video in the front end
+    displayVideo(videoId);
+
+    // checks to see if mock captions should be used
+    const queryParams = new URLSearchParams(window.location.search)
+    if (queryParams.has('mock')) {
+        sendJsonForm(JSON.stringify(MOCK_JSON_CAPTIONS));
+        return;
+    }
+    
+    // checks if mock nlp should be used
+    if (queryParams.has('mockall')) {
+        // Sets the results table
+        document.getElementById('output').innerHTML = styleEntitiesFromJson(MOCK_NLP_OUTPUT);
+
+        // clickable entities and timestamps
+        setClickableEntities();
+        sortEntities();
+        
+        return;
+    }
```

Comment:
are we planning on making mocking possible for anyone or are we going to take this out in the final product?

Reply:
I mainly included mocking so we don't need to deploy to see our UI changes. We can remove them in the final product if need be